src/gtk-3.0/gtk-widgets.css

```diff
@@ -532,39 +563,41 @@ list row:hover {
  * separators *
  **************/
 separator.horizontal {
-	border-image: url("assets/horizontal-line.png") 1 / 1px stretch;
+	border-image: url("assets/horizontal-line.svg") 1 / 1px stretch;
 	padding: 1px;
 }
 separator.vertical {
-	border-image: url("assets/vertical-line.png") 1 / 1px stretch;
+	border-image: url("assets/vertical-line.svg") 1 / 1px stretch;
 	padding: 1px;
 }
 
 /*********
  * paned *
  *********/
 paned > separator {
-	background: url("assets/paned-vertical-separator.png") no-repeat center;
+	background: url("assets/paned-separator-vertical.svg") no-repeat center;
 	-gtk-icon-source: none;
 	padding: 1px;
 }
 paned > separator:hover {
 	background-color: @theme_selected_bg_color;
 }
 paned.vertical > separator {
-	background-image: url("assets/paned-separator-horizontal.png");
+	background-image: url("assets/paned-separator-horizontal.svg");
 }
 
 /**********
  * frames *
  **********/
 frame > border {
-	border-image: url("assets/frame-etched-out.png") 2 / 2px stretch;
+	border: 1px solid white;
```

Comment:
Same issue I have in Flat-grey-rounded: frames have double borders now.

https://raw.githubusercontent.com/puppylinux-woof-CE/gtk3_polished_blue/master/screenshot.png
https://raw.githubusercontent.com/puppylinux-woof-CE/gtk3_polished_blue/eb95d3fb82d990f48386d76604e766f1005c4075/screenshot.png

Reply:
Using svg now